mreg_cli/main.py

```diff
@@ -165,7 +165,8 @@ def main():
         print(e)
         raise SystemExit() from None
     if args.show_token:
-        print(api.session.headers["Authorization"])
+        print(str(api.session.headers["Authorization"]).split(" ")[1])
```

Comment:
```suggestion
        print(str(api.session.headers.get("Authorization")).partition(" ")[1] or "Token not found")
```

A little bit of safety against weird headers.